.gitignore

```diff
@@ -1 +1,2 @@
 _site
+.DS_Store
```

Comment:
If you're adding a new line, it's good form to add the newline character after it too, so git tools won't complain like this.